src/mem/Pool.h

```diff
@@ -284,45 +279,23 @@ class MemImplementingAllocator : public MemAllocator
 class MemPoolStats
```

Comment:
The PR probably touches most of the lines that mention this class by name already. If so, please move this class into the Mem namespace.

Reply:
Done in [b06b4d9](https://github.com/squid-cache/squid/pull/1186/commits/b06b4d9064ecd1d9988818ada2e4fff970161701)